packages/plugin_platform_interface/README.md

```diff
@@ -45,8 +55,23 @@ code only to disable the `extends` enforcement.
 
 For example, a Mockito mock of a platform interface can be created with:
 
+<?code-excerpt "test/plugin_platform_interface_test.dart (Mock)"?>
 ```dart
-class UrlLauncherPlatformMock extends Mock
-   with MockPlatformInterfaceMixin
-   implements UrlLauncherPlatform {}
+class SamplePluginPlatformMock extends Mock
+    with MockPlatformInterfaceMixin
+    implements SamplePluginPlatform {}
 ```
+
+## A note about `base`
+
+In Dart 3, [the `base` keyword](https://dart.dev/language/class-modifiers#base)
+was introduced to the language, which allows enforcing that implementations
+of an interface use `extends` rather than `implements`. The Flutter team is
```

Comment:
>"which allows enforcing that implementations of an interface use" 

I'm not sure what this is supposed to say

Reply:
I adjusted the wording; hopefully it's clearer now.